src/Sharpliner/AzureDevOps/ConditionedExpressions/Condition.cs

```diff
@@ -66,6 +69,17 @@ public static string RemoveTags(string condition)
         return condition;
     }
 
+    protected static string WrapQuotes(string value)
```

Comment:
Can you use this method in `BranchCondition` too?

Reply:
BranchCondition inherits from EqualityCondition. I made that base class Inherit from String condition. 
What do you think?

It surfaced some existing tests that were asserting an incorrect syntax, so I've fixed those now too.

Reply:
Great, that is the best outcome